WNPRC_EHR/src/weight/typings/main.d.ts

```diff
@@ -0,0 +1,5 @@
+/* App globals */
```

Comment:
Is this file needed?

Reply:
Nope, this is probably not needed, it's a relic of some initial config :)